### PR TITLE
Add end2end tests for Next.js (standalone)

### PR DESCRIPTION
**Pull request by timokoessler**

### CODE REVIEW

File: library/sources/HTTPServer.test.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { wrap } from "../helpers/wrap";
-import * as pkg from "../helpers/isPackageInstalled";
```
Comment (hansott) [inline library/sources/HTTPServer.test.ts:2]: Remove isPackageInstalled helper?

Comment (timokoessler) [reply library/sources/HTTPServer.test.ts:2]: Done

File: sample-apps/nextjs-standalone/src/app/files/route.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  }
+
+  if (!stdout || stdout.length === 0) {
+    return NextResponse.json({ files: [] });
+  }
+
+  return NextResponse.json({
+    files: stdout.split("\n").filter((line) => line),
+  });
+}
```
Comment (hansott) [inline sample-apps/nextjs-standalone/src/app/files/route.js:36]: maybe good to add a route handler with a database query?

File: .github/workflows/end-to-end-tests.yml [hunk 3]
```diff
@@ -27,7 +27,7 @@ jobs:
         options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=5 -e MYSQL_ROOT_PASSWORD=mypassword -e MYSQL_DATABASE=catsdb --entrypoint sh mysql:8.0 -c "exec docker-entrypoint.sh mysqld --default-authentication-plugin=mysql_native_password"
         ports:
           - "27015:3306"
-    timeout-minutes: 5
+    timeout-minutes: 10
```
Comment (hansott) [inline .github/workflows/end-to-end-tests.yml:30]: Looks like we can keep the timeout at 5?

File: sample-apps/nextjs-standalone/src/app/files/route.js (hunk 2 above)
Comment (timokoessler) [reply sample-apps/nextjs-standalone/src/app/files/route.js:36]: In addition to or instead of Shell Injection?

**Pull request merged**